### DISCUSSION TITLE
Set duration in AUDIO sample

### DISCUSSION THREAD

**Discussion by toxeekmeelk** [aandrew-me/ytDownloader#200] [Ideas]
Hello,

I'm french so please excuse my possibly damaged English

I tried to set the duration of an audio sample (from a song) but it extracts the all track. Is there a way to extract only the selected part you want (by setting start and end) to make an audio sample as it seems to be possible with video ?

**Comment by aandrew-me** [comment]
For now its not possible, but if I may add in future
